stable/vpa/templates/admission-controller-deployment.yaml

```
@@ -26,6 +26,9 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.priorityClassName }}
+      priorityClassName:
```

I think you'll need to add the value too:

```suggestion
      priorityClassName: {{ . }}
```